.mvn/maven.config

```diff
@@ -1,4 +1,4 @@
--Drevision=1.8.0-SNAPSHOT
+-Drevision=1.10.0-SNAPSHOT
```

Comment:
Shouldn't it be 1.9?

Reply:
nope, we won't have a 1.9 for the server